.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,34 @@
-Please ensure the following is present in your PR:
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
That's fine, I just didn't see the text when viewing the `.md` file via the web UI.